addons/mail/models/mail_mail.py

```diff
@@ -80,6 +80,9 @@ def mark_outgoing(self):
     def cancel(self):
         return self.write({'state': 'cancel'})
 
+    def prepare_filtered_mails(self):
+        return self
```

Comment:
Document here that you can use filtered function for selecting records (because the usual form for this kind of hooks is to prepare the domain that it's going to be applied).